### PR TITLE
fix: i18n

### PR DESCRIPTION
**Pull request by AasmundN**
# Changes
- Switch to `pnpm` and update `dev-branch.yml` accordingly.
- Update _Nuxt_ from `3.5.2` to `3.6.5`.
- Remove detection of browser language. The default language should have priority. 

# TODO
- Update README to reflect change in package manager. This will be solved in #55 
- Maybe figure out why lazy loading isn't working. But I don't think this is critical.

### CODE REVIEW

**Comment by AasmundN** [comment]
The updates in the workflow need to be tested.

**Comment by AasmundN** [comment]
Hmm, also lazy loading of the locale files doesn't seem to be working. But if I set `lazy: false` everything works as expected.

File: .github/workflows/dev-branch.yml [hunk 1]
```diff
@@ -14,18 +14,23 @@ jobs:
...
+
       - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
           node-version: 18
+          cache: 'pnpm'
 
       - name: Install dependencies
-        run: npm install
+        run: pnpm install
```
Comment (ipeglin) [inline .github/workflows/dev-branch.yml:28]: Change to `pnpm clean install`

File: .github/workflows/dev-branch.yml [hunk 2]
```diff
@@ -38,13 +43,18 @@ jobs:
...
+
+      - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
           node-version: 18
+          cache: 'pnpm'
 
       - name: Install dependencies
-        run: npm install
+        run: pnpm install
```
Comment (ipeglin) [inline .github/workflows/dev-branch.yml:57]: `pnpm clean install`

File: nuxt.config.ts [hunk 3]
```diff
@@ -22,13 +22,13 @@ export default defineNuxtConfig({
...
-    '@pinia/nuxt',
-    'nuxt-vitest',
     '@nuxtjs/i18n',
+    '@pinia/nuxt',
     'vuetify-nuxt-module',
+    'nuxt-vitest',
   ],
   i18n: {
-    lazy: true,
+    lazy: false,
```
Comment (ipeglin) [inline nuxt.config.ts:31]: Hvorfor ikke lazy? Var det har bugen var?

File: pages/index.vue [hunk 4]
```diff
@@ -2,16 +2,24 @@
...
-    <VBtn v-else color="error" @click="signOut">Sign out</VBtn>
 
     <p>User from store: {{ user }}</p>
     <p>User from api: {{ data }} {{ error }}</p>
 
     <VBtn color="info" @click="createUser">Create user</VBtn>
     <VBtn color="error" @click="deleteUser">Delete user</VBtn>
     <VBtn color="success" @click="refresh">Refresh user data</VBtn>
+
+    <select v-model="locale" style="cursor: pointer">
```
Comment (ipeglin) [inline pages/index.vue:19]: Har et egen issue på å gjøre dette om til sin egen komponent, så slipper vi å hardkode inn over alt.

**Review by ipeglin — APPROVED** [review]
LGTM

File: nuxt.config.ts (hunk 3 above)
Comment (AasmundN) [reply nuxt.config.ts:31]: Yes, it's not working. Will look a little bit more at it.

File: pages/index.vue (hunk 4 above)
Comment (AasmundN) [reply pages/index.vue:19]: Yes, it should be removed. It's just there to make it easier for anyone checking this out to test that it's working.

**Pull request merged**